docs/docs/guides/intro_developers.md

```diff
@@ -0,0 +1,335 @@
+website directory.
+
+- `backend-dev`. It includes these containers:
+
+  - `db`, `redis`, `redis-insights`, `adminer`, `backend`
+
+- `ci`. It's basically both the backend and the frontend without the development
+  tools. It includes these containers:
+
+  - `db`, `redis`, `webdb`, `maildev`, `backend`, `web`
+
+- `inference`. It includes these containers:
+
+  - `inference-db`, `inference-server`, `inference-worker`, `inference-redis`
+
+- `observability`. It includes tools to monitor the application. It includes
+  these containers:
+  - `prometheus`, `grafana`, `netdata`
+
+Notice that you can combine profiles, for example, `ci` and `observability`.
+
+The following figure shows containers in circles, grouped in boxes that
+correspond with roles, not with actual profiles:
+
+```mermaid
+flowchart TD
+    browser[browser]
+    subgraph frontend
+        web(("web <br> (node.js)")) -->
+        webdb(("webdb <br> (Postgres)"))
+    end
+    subgraph backend
+        backend-c(("backend <br> (FastAPI)")) --> db(("db <br> (Postgres)"))
+        backend-c --> redis((redis))
+    end
+    subgraph observability
+        pro((prometheus))
+        grafana((grafana))
+        netdata((netdata))
+    end
+    subgraph ci
+        frontend
+        backend
+    end
+    subgraph inference
+        infworker1((inference-worker <br> 1))
+        infworker2((inference-worker <br> ...))
+        infworkerN((inference-worker <br> n))
+        infserv(("inference-server <br> (FastAPI)"))
+        infserv --> infdb(("inference-db <br> (Postgres)"))
+        infserv --> infredis(("inference-redis"))
+        infserv --> infworker1
+        infserv --> infworker2
+        infserv --> infworkerN
+    end
+    subgraph support[Dev support tools]
+        adminer((adminer))
+        redisinsights((redis-insights))
+        maildev((maildev))
+    end
+    web -- API calls --> backend-c
+    web -- API calls --> infserv
+    browser --> web
+```
+
+There are ansible playbooks in the
+[ansible](https://github.com/LAION-AI/Open-Assistant/tree/main/ansible)
+directory.
+
+In addition to the production deployment, available at the project website,
+<https://open-assistant.io/es>, there's also a staging deployment at
+<https://web.staging.open-assistant.io/>.
+
+## The teams
+
+The major contributors, founders and initiators are Yannic Kilcher
+([yk](https://github.com/yk) on GitHub), who has a
+[YouTube channel](https://www.youtube.com/@YannicKilcher) with a large
+community, [Andreas Köpf](https://github.com/andreaskoepf), who is the Tech
+Lead, [Christoph Schumann](https://github.com/christophschuhmann), who is the
+Organizational Lead and a founder of [LAION](https://laion.ai/), and Huu Nguyen,
+from [Ontocord](https://github.com/ontocord).
+
+There's a
+[CODEOWNERS](https://github.com/LAION-AI/Open-Assistant/blob/main/CODEOWNERS)
+file that lists the code owners of different parts of the project. It contains
+this information:
+
+```text
```

Comment:
I don't think we should copy the code owners file contents here, as the contents could change and then this would add another point to update. Just the link should be fine

Reply:
Makes sense. Fixed. Thank you!